### PR TITLE
Include basic arviz support 

### PR DESCRIPTION
**Pull request by hpesonen**
#### Summary:
Added arvix dependency and a method for converting `elfi.Sample` object to  `arviz.InferenceData`

#### Please make sure

- You have read [contribution guidelines](https://elfi.readthedocs.io/en/latest/developer/contributing.html)
- You have updated CHANGELOG.rst
- You have listed the copyright holder for the work you are submitting (see next section)

If your contribution adds, removes or somehow changes the functional behavior of the package, please check that

- You have included or updated all the relevant documentation, including docstrings
- You have added appropriate functional and unit tests to ensure the new features behave as expected
- You have run `make lint`, `make docs` and `make test`

and the proposed changes pass all unit tests (check step 6 of CONTRIBUTING.rst for details)

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD3 (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

File: elfi/methods/results.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import sys
 from collections import OrderedDict
 
+import arviz as az
```
Comment (uremes) [inline elfi/methods/results.py:11]: this import should be moved to the next block (there's an error when i run `make lint`)

**Review by uremes — CHANGES_REQUESTED** [review]
this looks good. just need to fix the style error that i mentioned in the previous comment

File: elfi/methods/results.py (hunk 1 above)
Comment (hpesonen) [reply elfi/methods/results.py:11]: I think I need to modify our test suite. Github tests should flag these as well.

Comment (hpesonen) [reply elfi/methods/results.py:11]: btw, I don't get an error here with `make lint`. And looking at it, shouldn't it be where it is?

Comment (uremes) [reply elfi/methods/results.py:11]: true and i don't get an error now either

**Review by uremes — APPROVED** [review]

**Pull request merged**